src/apis/fs.cpp

```diff
@@ -342,8 +349,9 @@ static int fs_copy(lua_State *L) {
             else if ((i == fromElems.size() - 1 && i == toElems.size() - 1)) err(L, 1, "Can't copy a directory inside itself");
         }
         if (equal) err(L, 1, "Can't copy a directory inside itself");
-        const auto retval = recursiveCopy(fromPath, toPath);
-        if (retval.first != 0) err(L, retval.first, retval.second.c_str());
+        std::error_code e;
+        fs::copy(fromPath, toPath, fs::copy_options::recursive, e);
```

Comment:
This doesn't handle creating all parent directories - make sure to do that first.